src/public/secure/admin.ejs

```diff
@@ -470,6 +508,22 @@
         loadDt();
         loadEventListeners();
     });
+
+    function addDetailInfo(heading, detail) {
+        const container = document.getElementById('fileDetailsModelInfoContent');
+        const rowDiv = document.createElement('div');
+        const headingDiv = document.createElement('div');
+        const detailDiv = document.createElement('div');
+        rowDiv.className = 'row';
```

Comment:
according to the documentation, a modal requires a `container-fluid` div in that a row is added, considering there will only ever be one `container-fluid`, i recommend added it into `modal-body` and making that new div `fileDetailsModelInfoContent`

Reply:
Implemented